synapse/cores/lmdb.py

```diff
@@ -671,3 +761,47 @@ def _subrangeRows(self, p_enc, first_val, last_val, limit, right_closed, do_coun
                 if limit is not None and count >= limit:
                     break
         return count if do_count_only else ret
+
+    def _getCoreType(self):
+        return 'lmdb'
+
+    def _getBlobValu(self, key):
+        key_byts = s_common.msgenpack(key.encode('utf-8'))
+        db = self._dbs.get(BLOB_STORE)
+        with self._getTxn() as txn:
+            cur = txn.cursor(db)  # type: lmdb.Cursor
+            ret = cur.get(key_byts, default=None)
+        return ret
+
+    def _setBlobValu(self, key, valu):
+        key_byts = s_common.msgenpack(key.encode('utf-8'))
+        db = self._dbs.get(BLOB_STORE)
+        with self._getTxn(write=True) as txn:
+            cur = txn.cursor(db)  # type: lmdb.Cursor
+            cur.put(key_byts, valu, overwrite=True)
+
+    def _hasBlobValu(self, key):
+        ret = self._getBlobValu(key)
+        if ret is None:
+            return False
+        return True
+
+    def _delBlobValu(self, key):
+        key_byts = s_common.msgenpack(key.encode('utf-8'))
+        db = self._dbs.get(BLOB_STORE)
+        with self._getTxn(write=True) as txn:
+            cur = txn.cursor(db)  # type: lmdb.Cursor
+            ret = cur.pop(key_byts)
```

Comment:
slightly better to use delete since you don't need the previous contents. (Also the same issue as above; just use `txn.delete`